File: TenMinsPy/day44.py
# Censor the word fuck and dead with constant length asterisk

class day44:
    def checkBadword(self, w):
        badwords = ['dead', 'fuck']
        for i in badwords:
            if i in w.lower():
                w = w.replace(i, "*"*len(i))
        print(w)

day = day44();
w = input("Enter a word\n")
day.checkBadword(w)

File: TenMinsJava/day44.java
// Censor the word fuck and dead with constant length asterisk
package TenMinsJava;

import java.util.Scanner;

public class day44 {
    public static void checkWord(String w) {
        String badword [] = {"fuck", "dead"};
        for (int i = 0; i < badword.length; i++) {
            if (w.toLowerCase().contains(badword[i])) {
            String word = "*".repeat(badword[i].length());
            w = w.replace(badword[i], word);
            }
        }
        System.out.println(w);
    }
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Enter a word");
        String inp = scan.nextLine();
        checkWord(inp);
    }
}
